tests/security/open_basedir_001.phpt

```diff
@@ -1,5 +1,5 @@
 --TEST--
-openbase_dir runtime tightning
+openbase_dir runtime tightening
```

Comment:
Not sure about this

Reply:
Could use "hardening" instead if that's what the test checks